app/controllers/oauth2_controller.rb

```diff
@@ -5,6 +5,8 @@
 # @!visibility private
 class Oauth2Controller < ApplicationController
 
+  force_ssl if: -> { RailsApiAuth.force_ssl }
```

Comment:
I just tried that and it doesn't work. According to the documentation, it has to be a proc or symbol representing the name of the method

https://github.com/rails/rails/blob/4a2c7c6e7ec88759ae5ac2489b093a2b8330809c/actionpack/lib/action_controller/metal/force_ssl.rb#L58